lib/api/src/tests/versions.test.js

```diff
@@ -239,5 +239,23 @@ describe('versions API', () => {
 
       expect(api.versionUpdateAvailable()).toEqual(true);
     });
+
+    it('new prerelease version', async () => {
```

Comment:
Will it work when the user is upgrading from one prerelease to another?

Reply:
added code & tests for this